py3status/modules/yandexdisk_status.py

```diff
@@ -33,33 +34,37 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 10
     format = 'Yandex.Disk: {status}'
+    string_busy = 'Busy'
+    string_off = 'Not started'
+    string_on = 'Idle'
 
-    def yadisk(self):
-        response = {'cached_until': self.py3.time_in(self.cache_timeout)}
+    def post_config_hook(self):
+        if not self.py3.check_commands(["yandex-disk"]):
+            return {'cached_until': self.py3.CACHE_FOREVER,
+                    'color': self.py3.COLOR_BAD,
+                    'full_text': string_unavailable}
```

Comment:
This is wrong.  `post_config_hook()` either returns nothing or an exception to disable the module.  And that needs a cleanup.